modules/ROOT/partials/configuration/spellchecker_dialog.adoc

```diff
@@ -2,8 +2,8 @@
 
 This option specifies the primary spell checking mode.
 
-* When set to `true`, the <<toolbarbuttons,`spellcheck` toolbar button>> will open a dialog that will step the user through each potential spelling error in the document.
-* When set to `false`, the <<toolbarbuttons,`spellcheck` toolbar button>> will enable or disable as-you-type spell checking and the <<toolbarbuttons,`spellcheckdialog` toolbar button>> will open the spell checker dialog.
+* When set to `true`, the xref:toolbarbuttons[`spellcheck` toolbar button] will open a dialog that will step the user through each potential spelling error in the document.
+* When set to `false`, the xref:toolbarbuttons[`spellcheck` toolbar button] will enable or disable as-you-type spell checking and the xref:toolbarbuttons[`spellcheckdialog` toolbar button] will open the spell checker dialog.
```

Comment:
```suggestion
* When set to `true`, the xref:plugins/premium/tinymcespellchecker.adoc#toolbarbuttons[`spellcheck` toolbar button] will open a dialog that will step the user through each potential spelling error in the document.
* When set to `false`, the xref:plugins/premium/tinymcespellchecker.adoc#toolbarbuttons[`spellcheck` toolbar button] will enable or disable as-you-type spell checking and the xref:plugins/premium/tinymcespellchecker.adoc#toolbarbuttons[`spellcheckdialog` toolbar button] will open the spell checker dialog.
```
`toolbarbuttons` doesn't exist in this file, so we need to make sure it links to where we expect.